### PR TITLE
bedtools GetFastaBed: add -nameOnly flag

### PR DESCRIPTION
**Pull request by FleurGaBru**
The "-nameOnly" flag differs from the "-name" flag by adding only the content of the 4th column in the bed file to the output fasta header. The "-name" flag adds both the name and the coordinates. Some tools require very specific requirements for the fasta header and there -nameOnly appears to be very convenient.

FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/bedtools/getfastaBed.xml [hunk 1]
```diff
@@ -61,13 +64,15 @@ $split
             <param name="fasta" value="nucBed1.fasta" ftype="fasta" />
             <param name="tab" value="False" />
             <param name="split" value="False" />
+            <param name="nameOnly" value="False" />
```
Comment (bernt-matthias) [inline tools/bedtools/getfastaBed.xml:67]: Would be cool to have one test with nameOnly true

Comment (FleurGaBru) [reply tools/bedtools/getfastaBed.xml:67]: Agree!

Comment (FleurGaBru) [reply tools/bedtools/getfastaBed.xml:67]: Can I change it without sending a new pull request? I am new here :)

Comment (bernt-matthias) [reply tools/bedtools/getfastaBed.xml:67]: > Can I change it without sending a new pull request? I am new here :)

Simply push to the branch in your fork. New commits will be added automatically to the PR.

File: tools/bedtools/getfastaBed.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="bedtools_getfastabed" name="bedtools GetFastaBed" version="@TOOL_VERSION@" profile="@PROFILE@">
+<tool id="bedtools_getfastabed" name="bedtools GetFastaBed" version="@TOOL_VERSION@+galaxy1" profile="@PROFILE@">
```
Comment (bernt-matthias) [inline tools/bedtools/getfastaBed.xml:1]:
```suggest
<tool id="bedtools_getfastabed" name="bedtools GetFasta" version="@TOOL_VERSION@+galaxy1" profile="@PROFILE@">
```

I would suggest to have the name identical to the actual (sub)command. As long as we do not change the id of the tool this should not have a negative side effect.

**Comment by bernt-matthias** [comment]
The changed test implies that the tests file getfastaBed_result2.tabular needs to be updated. The diff is

```diff
--- local_file
+++ history_data
@@ -1 +1 @@
-chr1:10-100	TTCTTACCTATTAGTGGTTGAACATCGTGATATGTATGTTGACGGCCATAAGGCTGCTTCTTGTTGTCGATAGAACTTCATGTGCCTGTA
+	TTCTTACCTATTAGTGGTTGAACATCGTGATATGTATGTTGACGGCCATAAGGCTGCTTCTTGTTGTCGATAGAACTTCATGTGCCTGTA
```

Even better would be to update the input fasta such that it has a header format that bedtools understand.

File: tools/bedtools/getfastaBed.xml (hunk 2 above)
Comment (FleurGaBru) [reply tools/bedtools/getfastaBed.xml:1]: I absolutely agree!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @FleurGaBru!

**Pull request merged**